relay-server/src/services/store.rs

```diff
@@ -302,10 +302,26 @@ impl StoreService {
                         .items()
                         .map(|item| item.ty().as_str())
                         .collect::<Vec<_>>();
+                    let attachment_types = envelope
+                        .items()
+                        .map(|item| {
```

Comment:
This looks like a `filter_map`?

Reply:
I want an entry for every envelope item, such that attachment_types and item_types have the same length. Ideally they would be in the same list, but hey, quick and dirty.